src/sqllogictest/src/runner.rs

```diff
@@ -202,25 +202,39 @@ impl fmt::Display for Outcome<'_> {
     }
 }
 
-#[derive(Default, Debug, Eq, PartialEq)]
-pub struct Outcomes([usize; NUM_OUTCOMES]);
+#[derive(Debug, Eq, PartialEq)]
+pub struct Outcomes {
+    inner: [usize; NUM_OUTCOMES],
+    skip_failures: bool,
+}
+
+impl Outcomes {
+    pub fn new(skip_failures: bool) -> Outcomes {
+        Outcomes {
+            skip_failures,
+            inner: Default::default(),
+        }
+    }
+}
 
 impl ops::AddAssign<Outcomes> for Outcomes {
     fn add_assign(&mut self, rhs: Outcomes) {
-        for (lhs, rhs) in self.0.iter_mut().zip(rhs.0.iter()) {
+        for (lhs, rhs) in self.inner.iter_mut().zip(rhs.inner.iter()) {
             *lhs += rhs
         }
     }
 }
 
 impl fmt::Display for Outcomes {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        let total: usize = self.0.iter().sum();
+        let total: usize = self.inner.iter().sum();
         write!(
             f,
             "{}:",
-            if self.0[SUCCESS_OUTCOME] == total {
+            if self.inner[SUCCESS_OUTCOME] == total {
                 "PASS"
+            } else if self.skip_failures {
+                "SKIP"
```

Comment:
Hrm, I don't think "SKIP" is the right terminology. The test didn't get skipped. It's just that the failure got ignored!

Reply:
Hm, yeah for some reason in my head this was the terminology that other systems use, but it definitely isn't.